main.yml

```diff
@@ -228,6 +228,21 @@ rhel:
       - id: remediations
   top_level: true
 
+ruledev:
+  title: Rule Development
+  channel: '#flip-mode-squad'
+  deployment_repo: https://github.com/RedHatInsights/ruledev-frontend-build
+  frontend:
+    title: Rule Development
+    paths:
+      - /ruledev
+    sub_apps:
+      - id: 'ruledev'
+        title: Home
+        default: true
+        group: ruledev
+  top_level: true
```

Comment:
`top_level` should only be defined if this is a bundle, or something at that level. I'm against that for this app, unless we hear from the higher-ups that `ruledev` needs to be its own separate category.